### PR TITLE
doc: Update getting started examples for .NET SDK

### PR DESCRIPTION
**Pull request by adityaoberai**
## What does this PR do?

Update the getting started example for .NET:
- Align code to C# coding conventions
- Add missing using directives
- Correct playground link title

## Related PRs and Issues

- No

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: docs/sdks/dotnet/GETTING_STARTED.md [hunk 1]
```diff
@@ -1,44 +1,47 @@
...
 ```csharp
 using Appwrite;
+using Appwrite.Services;
+using Appwrite.Models;
 
-static async Task Main(string[] args)
-{
-  var client = Client();
+var client = new Client()
+  .SetEndpoint("http://cloud.appwrite.io/v1")  // Make sure your endpoint is accessible
```
Comment (gewenyu99) [inline docs/sdks/dotnet/GETTING_STARTED.md:12]: You don't need to make sure endpoint is accessible anymore ;) It's cloud

File: docs/sdks/dotnet/GETTING_STARTED.md [hunk 2]
```diff
@@ -1,44 +1,47 @@
...
+  .SetProject("5ff3379a01d25")                 // Your project ID
+  .SetKey("cd868db89")                         // Your secret API key
+  .SetSelfSigned();                            // Use only on dev mode with a self-signed SSL cert
```
Comment (gewenyu99) [inline docs/sdks/dotnet/GETTING_STARTED.md:15]: I don't think you should have this here if it's pointed to cloud.

**Review by gewenyu99 — COMMENTED** [review]
Left some comments.

File: docs/sdks/dotnet/GETTING_STARTED.md (hunk 1 above)
Comment (adityaoberai) [reply docs/sdks/dotnet/GETTING_STARTED.md:12]: Might still be a good note for people who replace it with self-hosted links :)

File: docs/sdks/dotnet/GETTING_STARTED.md (hunk 2 above)
Comment (adityaoberai) [reply docs/sdks/dotnet/GETTING_STARTED.md:15]: Should we do that, or maybe we can update the comment to mention that it should only be used if the user has a self-hosted instance without a domain pointed to it?

Comment (gewenyu99) [reply docs/sdks/dotnet/GETTING_STARTED.md:15]: Our attitude at the moment is cloud first, so make this copy pasteable for cloud.

Maybe, we can split this into cloud and self-hosted (we should do for all SDKs) but you can leave a comment in experience channel later and I'll make it happen following ur example here ;)

Comment (adityaoberai) [reply docs/sdks/dotnet/GETTING_STARTED.md]: Made the update :)

**Review by christyjacob4 — APPROVED** [review]
Can you also update the sdk version in the platforms.php file ?

**Comment by adityaoberai** [comment]
> Can you also update the sdk version in the platforms.php file ?

Updated 👍

**Comment by christyjacob4** [comment]
The SDK generator version also needs to be updated in the composer and composer-lock files @adityaoberai

**Comment by adityaoberai** [comment]
@christyjacob4 updated those too

**Review by christyjacob4 — APPROVED** [review]

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**